app/styles/player.styl

```diff
@@ -226,20 +226,35 @@
             display: none
 
         #volume-bar
+            position: absolute
             width: 90px
+            height: 100%
+            border-radius: 2px
+            display: inline-block
+            cursor: pointer
+
+        #volume-bar::before,
+        #volume
+            position: absolute
+            content: ""
```

Comment:
Prefer to split rules that doesn't apply to an element in the selectors. Here, `content` and `background-color` doesn't apply to `#volume` so I suggest we put them on a `#volume::before` selector ruleset below